action-api/src/test/java/com/powsybl/action/json/JsonActionTest.java

```diff
@@ -83,6 +83,7 @@ void actionRoundTrip() throws IOException {
                 .withStaticVarCompensatorId("svc").withRegulationMode(StaticVarCompensator.RegulationMode.REACTIVE_POWER)
                 .withReactivePowerSetpoint(120.0).build());
         actions.add(new TerminalsConnectionAction("id4", "transformerId25", ThreeSides.THREE, true)); // only one side.
+        actions.add(new AreaInterchangeTargetUpdateAction("id99", "AreaA", 101.0));
```

Comment:
please add also a test with a target set to NaN (= remove the target), this should be supported.

Reply:
NaN is already correctly interpreted on AIC side ?

Reply:
yes in OLF